app/services/setting_registry.rb

```diff
@@ -42,6 +57,15 @@ def []=(name, value)
     definition.value = db_record.value
   end
 
+  # Returns all the categories used for settings
+  def categories
+    sticked_general = ActiveSupport::OrderedHash.new
```

Comment:
https://api.rubyonrails.org/classes/ActiveSupport/OrderedHash.html mentions it's deprecated. I guess that's because Hash always behaves this way now? Quoting https://ruby-doc.org/core-2.5.0/Hash.html:
> Hashes enumerate their values in the order that the corresponding keys were inserted.

Reply:
I've noticed the deprecation, but been wondering why that is... cool, in that case, simple hash is enough :+1: